addons/rose/tests/integration/components/rose/table/row-test.js

```diff
@@ -7,17 +7,20 @@ module('Integration | Component | rose/table/row', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders', async function (assert) {
+    assert.expect(2);
```

Comment:
In this test file I added assert expectations for each test. I do not know how strict we want to be with the assert expectation. If we want be more relaxed I can delete it.

Reply:
I think these are very useful especially for async tests.